pages/pay-fees.html

```diff
@@ -35,10 +35,11 @@ <h1>Pay Fees</h1>
                     <li><a href="https://buy.stripe.com/fZe7tDcx6eno5C8dQQ">$40 Annual mebership fee</a></li>
                     <li><a href="https://buy.stripe.com/eVaaFP1Sscfg6Gc001" title="Click to buy a single sail">$20 Single sail</a</li>
                     <li><a href="https://buy.stripe.com/9AQ3dn40A7Z06Gc7su" title="Click to buy 10 sails page">$150 10 sails package</a></li>
-                    <li><a href="https://buy.stripe.com/cN25lv8gQgvwe8E147" title="Click to buy unlimited sails package">$210 Unlimited sails package</a></li>
+                    <li><a href="https://buy.stripe.com/8wM8xH68Ifrse8E7sA" title="Click to buy annual membership and 10 sails page">$190 Annual membership and 10 sails package</a></li>
+                    <li><a href="https://buy.stripe.com/3cs6pzdBa9341lSfZ5" title="Click to buy annual membership and unlimited sails package">$250 Annual membership and unlimited sails package</a></li>
                     <li><a href="https://buy.stripe.com/3cs01bgNmfrs1lSbIM" title="Click to buy guest sail">$25 Guest sail fee</a></li>
                 </ul>
-                <p>You can also pay the fees via INTERACT e-Transfer to our finance email.</p>
+                <p>You can also pay the fees via INTERACT e-Transfer to our finance email: <a href=finance@blindsailing.ca>finance@blindsailing.ca.</a></p>
```

Comment:
finance will start getting spam emails from web scrapers. are you sure they want that?